Geometry/HcalCommonData/python/testPhase1GeometryXML_cfi.py

```diff
@@ -224,8 +224,8 @@
         'Geometry/HcalCommonData/data/hcalouteralgo.xml',
         'Geometry/HcalCommonData/data/hcalforwardalgo.xml',
         'Geometry/HcalCommonData/data/average/hcalforwardmaterial.xml',
-        'Geometry/HcalCommonData/data/hcalSimNumbering/2019/hcalSimNumbering.xml', 
-        'Geometry/HcalCommonData/data/hcalRecNumbering/2019/hcalRecNumbering.xml', 
+        'Geometry/HcalCommonData/data/hcalSimNumbering/2021/hcalSimNumbering.xml', 
```

Comment:
@boudoul - a Phase1 configuration using something from the future 2021 seem to be confusing...

Reply:
2021 will be the first year that the *complete* Phase 1 upgrade will be used for data taking.